src/sentry/snuba/query_subscription_consumer.py

```diff
@@ -251,12 +270,13 @@ def process_message(message: Message[KafkaPayload]) -> None:
                 offset = value.offset
                 partition = value.partition.index
                 message_value = value.payload.value
+                logical_topic = dataset_to_logical_topic[self.dataset]
                 try:
                     handle_message(
                         message_value,
                         offset,
                         partition,
-                        self.topic,
+                        logical_topic,
```

Comment:
Isn't this function expecting the real topic value used in prod not the "logical topic"?